hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -282,7 +281,12 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
             database,
             tableName,
             e);
-        commitStatus = checkCommitStatus(newMetadataLocation, metadata);
+        commitStatus =
```

Comment:
Here we can use the original method from `BaseMetastoreTableOperations.checkCommitStatus`  too.